.github/workflows/release.yml

```
@@ -46,7 +46,11 @@ jobs:
           changelog-types: ${{ steps.configure-changelog.outputs.result }}
           token: ${{ steps.generate_token.outputs.token }}
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2'
```

```suggestion
      - uses: actions/checkout@v2
```
think shouldn't be there